e2e/tests/notebook/import-r-lib.spec.ts

```diff
@@ -0,0 +1,84 @@
+import DataPage, {TabLabelAlias} from 'app/page/data-page';
+import {Language} from 'app/text-labels';
+import * as fs from 'fs';
+import {config} from 'resources/workbench-config';
+import {makeRandomName} from 'utils/str-utils';
+import {findWorkspace, signIn} from 'utils/test-utils';
+import {CellType} from 'app/page/notebook-cell';
+
+// Notebook server start may take a long time. Set maximum test running time to 20 minutes.
+jest.setTimeout(20 * 60 * 1000);
+
+describe('Import R libraries', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+  test('Print sys environment details', async () => {
+    let rCodeSnippet;
+    try {
+      rCodeSnippet = fs.readFileSync('resources/r-code/sys-print.txt', 'utf8');
+    } catch(e) {
+      throw new Error(e);
+    }
+
+    const workspaceCard = await findWorkspace(page);
+    await workspaceCard.clickWorkspaceName();
+
+    const dataPage = new DataPage(page);
+    const notebookName = makeRandomName('print-sys');
+    const notebook = await dataPage.createNotebook(notebookName, Language.R);
+
+    const cellIndex = 1;
+    const outputText = await notebook.runCodeCell(cellIndex, {code: rCodeSnippet});
+    console.log(`Code cell [${cellIndex}] output: \n${outputText}`);
+
+    // Delete notebook
+    const analysisPage = await notebook.goAnalysisPage();
+    await dataPage.openTab(TabLabelAlias.Analysis);
+    await analysisPage.deleteNotebook(notebookName);
+
+    // UserEmail and Google-bucket is expected to be printed out.
+    const email = config.userEmail
+    expect(outputText).toEqual(expect.stringMatching(email));
+    expect(outputText).toEqual(expect.stringMatching('gs://fc-secure-'))
```

Comment:
assertions are done after delete notebooks. this is to make sure delete always happen even if assertion fail.
this new pattern is applied in other notebook tests as well.